test/powershell/Modules/Microsoft.PowerShell.Core/RemoteGetModule.Tests.ps1

```diff
@@ -0,0 +1,80 @@
+Describe "Remote module tests" -Tags 'Feature','RequireAdminOnWindows' {
+
+    BeforeAll {
+        $originalDefaultParameterValues = $PSDefaultParameterValues.Clone()
+        if (!$IsWindows) {
+            $PSDefaultParameterValues["it:skip"] = $true
+        } else {
+            $pssession = New-RemoteSession
+            $cimsession = New-RemoteSession -CimSession
+        }
+    }
+
+    AfterAll {
```

Comment:
Edge case (minor): Last time I checked, afterAll doesn't run if you hit control-c or some other error fatal to pester happens while running these tests.  The Try-Finally pattern might be better for setting up this parameter to ensure that it always gets removed.

Reply:
We've been using this pattern everywhere.  If there is a case where AfterAll/AfterEach doesn't get run, we should open an issue for Pester